custom_components/elastic/index_mapping.json

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
This only includes a subset of ECS fields (as specified [here](https://github.com/elastic/ecs/blob/v1.0.0/generated/elasticsearch/6/template.json)). I was torn between including all ECS fields, or just a subset. I opted for the latter to make it a little easier to see what we're using (or potentially using). The downside with this approach is that adding more fields will require the index template to be updated which will not cascade to existing indices without a reindex operation, which is something this component isn't supporting at the moment.

I'm open to revisiting this decision